front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -1,23 +1,24 @@
 import React, { useEffect, useState, useRef, useMemo } from 'react';
 
 import { ChevronLeft, ChevronRight } from '@osrd-project/ui-icons';
+import { Manchette } from '@osrd-project/ui-manchette';
 import cx from 'classnames';
 import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 import { Rnd } from 'react-rnd';
 
 import useSimulationResults from 'applications/operationalStudies/hooks/useSimulationResults';
 import type { TrainSpaceTimeData } from 'applications/operationalStudies/types';
+import type { ProjectPathTrainResult } from 'common/api/generatedEditoastApi';
```

Comment:
Use osrdEditoastApi and not generatedEditoastApi
```suggestion
import type { ProjectPathTrainResult } from 'common/api/osrdEditoastApi';
```

Reply:
done